cpp/src/arrow/compute/kernels/scalar_if_else_benchmark.cc

```diff
@@ -85,11 +121,25 @@ static void IfElseBench(benchmark::State& state) {
                           (GetBytesProcessed<BooleanType>::Get(cond) +
                            GetBytesProcessed<Type>::Get(left) +
                            GetBytesProcessed<Type>::Get(right)));
+  state.SetItemsProcessed(state.iterations() * (len - offset));
```

Comment:
How about we also make the parameters more readable?
```suggestion
  state.SetItemsProcessed(state.iterations() * (len - offset));
  state.counters["offset"] = static_cast<double>(offset);
```